### DISCUSSION TITLE
Keychain Access Groups

### DISCUSSION THREAD

**Discussion by below** [microsoftconnect/ms-intune-app-sdk-ios#462] [Q&A] [answered]
As per the [Intune SDK for iOS Documentaion](https://learn.microsoft.com/en-us/mem/intune/developer/app-sdk-ios-phase3), the target apps needs the following Keychain Access Groups:

* `YOURBUNDLESEEDID.*`
* `com.microsoft.intune.mam`
* `com.microsoft.adalcache`

We are building several variants of our App for various test environments. Will these Apps be able to co-exist on the same device, or will the Intune SDK re-use the Keychain Storage for these Apps and thus create side effects?

**Comment by wangxiaoms** [comment, marked as answer]
Applications built by the same Apple developer team can co-exist on the same device. The keychain sharing access groups you mentioned should not cause any side effects. If you encounter any issue related to it let us know.
